### ISSUE TITLE
[Feature Request] Increment by more than one day at a time for charts

### ISSUE THREAD

**Issue by Knicey** [Lacro59/playnite-gameactivity-plugin#52]
I'd like to be able to increment by more than one day at a time for game session charts

Currently, I am only able to shift the graph by one day at a time for each press of the button (left or right)

For example, let's say this is what the graph looks like right now
A
![image](https://user-images.githubusercontent.com/45131167/140615251-648284be-6132-4c5f-be42-8960ec544134.png)

One left button press results in this (shifts by 1 day)
B
![image](https://user-images.githubusercontent.com/45131167/140615280-0a0943bf-ca46-4e3d-b678-4ba2b8c759fa.png)

Instead, I'd like it to come to this (shifts by 7 days)
C
![image](https://user-images.githubusercontent.com/45131167/140615347-22b97c79-bb59-4c3c-be39-df53abc04b4e.png)

The 7 days is just an example, preferably it would be a customizable number of shifts

In summary -
Right now the charts work like A to B
But I'd like the functionality to make it go from A to C
